src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkParsing.ts

```diff
@@ -117,6 +117,8 @@ function generateLinkSuffixRegex(eolOnly: boolean) {
 		// foo: (339)
 		//   ...
 		`:? ?[\\[\\(]${r()}(?:, ?${c()})?[\\]\\)]` + eolSuffix,
+		// Support for '/path/to/file, 204'
+		`['"]?${p()}, ${r()}` + eolSuffix,
```

Comment:
I think we could add this to the top entry in the array, by just adding `,` as an option in this group:

```
Before: (?::|#| |['"],)
 After: (?::|#|,| |['"],)
```

We will also need some tests to cover this case here:

https://github.com/microsoft/vscode/blob/13ac4f2de288322b1fd31415e3a0b8c9bc43ea07/src/vs/workbench/contrib/terminalContrib/links/test/browser/terminalLinkParsing.test.ts#L42-L62